torch/csrc/onnx/onnx.cpp

```diff
@@ -23,6 +23,12 @@ bool micropb_encode<double, nullptr>(pb_ostream_t *stream, double* arg) {
   return pb_encode_fixed64(stream, static_cast<void*>(arg));
 }
 
+template <>
+bool micropb_encode<Dimension, nullptr>(pb_ostream_t *stream, Dimension* arg) {
```

Comment:
When I wrote `micropb_encode`, the original intent was that it covered exactly "native" protobuf types; anything non-standard would get a `MicroProto` class. Was there something that made adding `Dimension` here much more convenient? Curious.

Reply:
Dimension is kind of special, since it only has one boolean and one int64 member (Actually there is another callback member, but right now we never use it). So I treat it like DataType. Also it is not ending with Proto... So I decide to let it act like a basic type.

Reply:
OK, I filed an issue about this here: https://github.com/onnx/onnx/issues/78